sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc

```diff
@@ -934,6 +934,9 @@ _Returns (1):_ The value `true` only if the device's architecture is equal to
 
 _Returns (2):_ The value `true` only if the device's architecture is in the
 category `category`.
+
+If current device architecture is not supported by the extension, a
+`sycl::exception` with error code `sycl::errc::runtime` will be thrown.
```

Comment:
Does this happen when the device has an architecture that is not in the `enum class architecture` enumeration?  I think I'd rather return `false` in this case, rather than throwing an exception.

Reply:
Good point! I will remove this from spec and update impl to reflect that